### DISCUSSION TITLE
Problem in providing the proof for Student Developer Pack

### DISCUSSION THREAD

**Discussion by ismayilovh** [community/community#65850] [GitHub Education]
### Hi everyone,

I am a student at Lyceum 27, Ganja, Azerbaijan. I'm 15 years old, and I have been trying to apply for the GitHub Student Developer Pack. Unfortunately, my application was rejected 12 times, although I uploaded several different documents. The reason is this:

"The item you uploaded is insufficient to demonstrate your current academic status.
Please see suggestions.
Suggested proof to expedite your approval:

You are unlikely to be verified until you have completed your GitHub user profile with your full name exactly as it appears in your academic affiliation document plus a short bio. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.

Have you completed your GitHub user profile with all your relevant information, such as your full name as it appears in your image and a short bio?"

I'm sure that my GitHub full name is same with the name on the document. Can someone say how can I solve this problem? Thanks a lot!

**Comment by hkfmz** [comment]
Make sure your full name on GitHub matches exactly with your academic document, without any variations. Add a brief and relevant biography to your GitHub profile. Then, log out and log back in to GitHub before resubmitting your request for the GitHub Student Developer Pack. This should improve your chances of approval. If the issue persists, reach out to GitHub support for assistance. Good luck!

**Reply by MohammadMDSA** [reply]
Hi, I have the same problem. My name on my student ID is in Persian (should I change my profile name to the Persian version?). Also, the date on my card is in Persian Calendar. Another barrier is that the name and logo of the university are not on the same side of the card where the date is. What should I do?

**Reply by ismayilovh** [reply]
Thanks for your advice. I made so many variations (22+ variations), with the email government gave me and with my own gmail, but all of them are rejected.

**Reply by MohammadMDSA** [reply]
I realized my mistake. I made a jpg file containing 3 photos. Front and back of my card and another one for translation of the labels, numbers and dates.

**Reply by ismayilovh** [reply]
> I realized my mistake. I made a jpg file containing 3 photos. Front and back of my card and another one for translation of the labels, numbers and dates.

I understood, did you get approved?

**Reply by MohammadMDSA** [reply]
I did

**Reply by ismayilovh** [reply]
> I did

Thanks for your reply. I'm a high school student, and I don't have a student card, but I have many documents like honor certificates, thank-you letters, etc. that prove I'm studying at that school. Can you please say what I can do with those documents? Do I need an English-translated version of one of those documents? Thanks in advance.

**Reply by MohammadMDSA** [reply]
I suggest you grab the document stating you are a student until a specific date. Then have it translated with Google Translate, so the vision module would recognize your name and school name from the translation. Put these images together with Paint, and reapply.

**Reply by ismayilovh** [reply]
> I suggest you grab the document stating you are a student until a specific date. Then have it translated with Google Translate, so the vision module would recognize your name and school name from the translation. Put these images together with Paint, and reapply.

Thanks a lot for your recommendation. I will follow your advice (if I find a document that has that specific date :'

**Reply by MohammadMDSA** [reply]
I believe you can request a formal one from your school if you don't have any.
Happy to help!

**Reply by ismayilovh** [reply]
> I believe you can request a formal one from your school if you don't have any.
> Happy to help!

Yes, you're right. Thanks a lot!
